.changes/unreleased/Breaking-20241111-195013.yaml

```
@@ -0,0 +1,12 @@
+kind: Breaking
+body: |
+AsService now runs "DefaultCmd" by default instead of last "Exec" command.
+
+User can override the args by providing "ContainerAsServiceOpts.Args" option. They can also 
+configure the container to use Entrypoint by using "ContainerAsServiceOpts.UseEntrypoint" option. 
+
+See sdk documentation for additional options.
```

This isn't valid yaml is it?
```suggestion
    AsService now runs "DefaultCmd" by default instead of last "Exec" command.

    User can override the args by providing "ContainerAsServiceOpts.Args" option. They can also 
    configure the container to use Entrypoint by using "ContainerAsServiceOpts.UseEntrypoint" option. 

    See sdk documentation for additional options.
```